crates/handlers/src/admin/v1/users/deactivate.rs

```diff
@@ -49,7 +51,25 @@ impl IntoResponse for RouteError {
     }
 }
 
-pub fn doc(operation: TransformOperation) -> TransformOperation {
+/// # JSON payload for the `POST /api/admin/v1/users/:id/deactivate` endpoint
+#[derive(Default, Deserialize, JsonSchema)]
+#[serde(rename = "DeactivateUserRequest")]
+pub struct Request {
+    /// Whether to skip locking the user before deactivation.
+    #[serde(default)]
+    skip_lock: bool,
```

Comment:
If this is to keep the previous behaviour by default, I'm fine with breaking the API and not have it lock

Reply:
Thanks for the go-ahead on API-breaking changes.  So I'll opt for making {d,r}eactivate and {,un}lock do only their namesake action.